### PR TITLE
Ft/set sproxyd path

### PR DESCRIPTION
**Pull request by dora-korpar**
Gives user ability to set custom sproxyd path in bootstrap array

### CODE REVIEW

File: tests/unit/sproxyd.js [hunk 1]
```diff
@@ -92,10 +103,11 @@ function handler(req, res) {
     }
 }
 
+const clientCustomPath =
+    new Sproxy({ bootstrap: ['127.0.0.1:9001:/proxy/arc/'] });
```
Comment (RachedBenMustapha) [inline tests/unit/sproxyd.js]: Why the second `:` here?

**Comment by dora-korpar** [comment]
@ironman-machine try

File: tests/unit/sproxyd.js (hunk 1 above)
Comment (dora-korpar) [inline tests/unit/sproxyd.js]: To make sure the customizable path feature gets parsed correctly :)

**Comment by dora-korpar** [comment]
@ironman-machine try

File: tests/unit/sproxyd.js (hunk 1 above)
Comment (bennettbuchanan) [inline tests/unit/sproxyd.js]: This does seem to differ from the intended use of the parsing method (https://github.com/scality/sproxydclient/blob/master/lib/sproxyd.js#L39). But I trust this is the best approach here.

**Comment by dora-korpar** [comment]
@ironman-machine try
SCALITY_S3_BRANCH=ft/setSproxydPath

**Pull request merged**